src/libraries/apicompat/ApiCompatBaseline.NetCoreAppLatestStable.txt

```diff
@@ -131,6 +137,10 @@ Compat issues with assembly System:
 CannotChangeAttribute : Attribute 'System.Runtime.Versioning.SupportedOSPlatformAttribute' on 'System.Diagnostics.Process.MaxWorkingSet.set(System.IntPtr)' changed from '[SupportedOSPlatformAttribute("freebsd")]' in the contract to '[SupportedOSPlatformAttribute("freebsd")]' in the implementation.
 CannotChangeAttribute : Attribute 'System.Runtime.Versioning.SupportedOSPlatformAttribute' on 'System.Diagnostics.Process.MinWorkingSet.set(System.IntPtr)' changed from '[SupportedOSPlatformAttribute("freebsd")]' in the contract to '[SupportedOSPlatformAttribute("freebsd")]' in the implementation.
 CannotChangeAttribute : Attribute 'System.Runtime.Versioning.UnsupportedOSPlatformAttribute' on 'System.Security.Cryptography.X509Certificates.PublicKey.GetDSAPublicKey()' changed from '[UnsupportedOSPlatformAttribute("ios")]' in the contract to '[UnsupportedOSPlatformAttribute("browser")]' in the implementation.
+Compat issues with assembly System.Console:
+CannotChangeAttribute : Attribute 'System.Runtime.Versioning.SupportedOSPlatformAttribute' on 'System.Console.SetWindowSize(System.Int32, System.Int32)' changed from '[SupportedOSPlatformAttribute("windows")]' in the contract to '[SupportedOSPlatformAttribute("linux")]' in the implementation.
+CannotChangeAttribute : Attribute 'System.Runtime.Versioning.SupportedOSPlatformAttribute' on 'System.Console.WindowHeight.set(System.Int32)' changed from '[SupportedOSPlatformAttribute("windows")]' in the contract to '[SupportedOSPlatformAttribute("linux")]' in the implementation.
+CannotChangeAttribute : Attribute 'System.Runtime.Versioning.SupportedOSPlatformAttribute' on 'System.Console.WindowWidth.set(System.Int32)' changed from '[SupportedOSPlatformAttribute("windows")]' in the contract to '[SupportedOSPlatformAttribute("linux")]' in the implementation.
```

Comment:
@ViktorHofer, is there a bug in the compat tooling, or is something wrong in the implementation in this PR?  This text says `changed from '[SupportedOSPlatformAttribute("windows")]' in the contract to '[SupportedOSPlatformAttribute("linux")]' in the implementation`, but shouldn't it say that it changed to be both windows and linux, not just linux?

Reply:
I defer to @smasher164 who just recently reimplemented this rule in the APICompat tooling. @smasher164 I think you noticed that the logging in the old APICompat is misleading. As we will soon switch to the new tooling, has this situation gotten better now?

Reply:
@smasher164 Would you comment on this? Is this OK for this PR?

Reply:
The existing tooling is a bit off when it comes to these messages but that shouldn't block this PR. We will get this cleaned-up when we switch to the new roslyn based APICompat tooling.